services/newsfeed/shared/schema/newsfeed.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
 		image_url: { type: 'string' },
 		source: { type: 'string', null: false },
 		source_id: { type: 'string' },
-		ctime: { type: 'integer', null: false },
-		mtime: { type: 'integer' },
+		createdAt: { type: 'integer', null: false },
+		modifiedAt: { type: 'integer' },
```

Comment:
Kindly consider setting `modifiedAt` to be `non-nullable`. (The values should be defaulting to the corresponding `createdAt` values)
If not, please consider updating the Joi schema for the newsfeed articles in the below files to be `optional()`:

- services/newsfeed/tests/schemas/newsfeedArticle.schema.js
- tests/schemas/newsfeed.schema.js